build.sbt

```diff
@@ -2,7 +2,21 @@ name := "compiler-benchmark"
 
 version := "1.0-SNAPSHOT"
 
-scalaVersion in ThisBuild := "2.11.8"
+def scala211 = "2.11.11"
+def dottyLatest = "0.2.0-RC1"
+scalaVersion in ThisBuild := scala211
+
+commands += Command.command("testAll") { s =>
+  "test:compile" ::
+    "compilation/test" ::
+    "hot -psource=scalap -wi 1 -i 1 -f1" ::
```

Comment:
Note. I bumped down the warmup iterations from 10 to 1 to make the tests run faster. The benchmark numbers on Travis are too unreliable anyways.